pkgs/applications/blockchains/bitcoin/default.nix

```diff
@@ -77,39 +76,38 @@ stdenv.mkDerivation (finalAttrs: {
 
   postInstall =
     ''
-      installShellCompletion --bash contrib/completions/bash/bitcoin-cli.bash
```

Comment:
maybe just do `cd ..` as the first step and then you do not have to change all the lines below